### PR TITLE
Image index

### PR DESCRIPTION
**Pull request by kailanjian**

### CODE REVIEW

**Comment by kailanjian** [comment]
[Issue 3: Image Index](https://trello.com/c/L87nafQR/223-issue-3-image-index)

File: app/controllers/welcome_controller.rb [hunk 1]
```diff
@@ -1,3 +1,5 @@
 class WelcomeController < ApplicationController
-  def home; end
+  def home
```
Comment (darreneng) [inline app/controllers/welcome_controller.rb]: We should put this on the images controller and then point the root route to the index action

File: app/controllers/welcome_controller.rb [hunk 2]
```diff
@@ -1,3 +1,5 @@
...
+    @images = Image.all.sort_by(&:created_at).reverse
```
Comment (darreneng) [inline app/controllers/welcome_controller.rb]: we don't need to reverse this in memory 

look at `sort`

File: test/controllers/welcome_controller_test.rb [hunk 3]
```diff
@@ -1,16 +1,29 @@
...
+    assert_response :success
     assert_select 'h1', 'Hello, world'
     assert_select 'a' do
       assert_select '[href=?]', new_image_path
     end
   end
+
+  def test_home__db_images
+    Image.create(link: 'http://www.image1.com')
+    Image.create(link: 'http://www.image2.com')
```
Comment (darreneng) [inline test/controllers/welcome_controller_test.rb]: opt for using 3 images

**Pull request merged**